docs/env.md

```diff
@@ -4,10 +4,28 @@ Unless specified otherwise, a variable is enabled if and only if it's set to `1`
 
 ### DRM
 
-`AQ_DRM_DEVICES` -> Set an explicit list of DRM devices (GPUs) to use. It's a colon-separated list of paths, with the first being the primary. E.g. `/dev/dri/card1:/dev/dri/card0`
-`AQ_NO_ATOMIC` -> Disables drm atomic modesetting
-`AQ_MGPU_NO_EXPLICIT` -> Disables explicit syncing on mgpu buffers
-`AQ_NO_MODIFIERS` -> Disables modifiers for DRM buffers
+### DRM Devices and Settings
+
+`AQ_DRM_DEVICES` -> This option allows you to set a specific list of DRM devices (GPUs) for the system to use.  
+  The list is separated by colons (:), and the first device listed will be the primary GPU.  
+  For example: `/dev/dri/card1:/dev/dri/card0`.  
+  This means the system will primarily use `card1` and then fall back to `card0` if needed.
```

Comment:
May be good to add a part here about symlinking to `~/.config/hypr/card`. There was a good section about that in the pre-aq docs.